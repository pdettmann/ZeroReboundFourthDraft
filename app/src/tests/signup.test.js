// import React from 'react';
// import Signup from '../signup';
// import { configure, shallow } from 'enzyme';
// import Adapter from 'enzyme-adapter-react-16';

// describe('Article component', () => {
//     // it('tests something ', () => {
//     //     const wrapper = shallow(<Signup/>);
//     //     const signupState = wrapper.state().firstName;
//     //     expect(signupState).toEqual(undefined);
//     // });
//   });
