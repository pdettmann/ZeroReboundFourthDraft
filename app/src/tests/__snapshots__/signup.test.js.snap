// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup renders correctly 1`] = `
<div
  className="signup"
>
  <button
    onClick={[Function]}
  >
    Sign in
  </button>
  <h3>
    ZeroRebound
  </h3>
  <h1>
    Register
  </h1>
  <input
    id="fname"
    onChange={[Function]}
    placeholder="First Name"
    type="text"
  />
  <br />
  <input
    id="lname"
    onChange={[Function]}
    placeholder="Last Name"
    type="text"
  />
  <br />
  <input
    id="email"
    onChange={[Function]}
    placeholder="Email"
    type="text"
  />
  <br />
  <input
    id="password"
    onChange={[Function]}
    placeholder="Password"
    type="password"
  />
  <br />
  <input
    id="passwordTwo"
    onChange={[Function]}
    placeholder="Repeat Password"
    type="password"
  />
  <br />
  <button
    onClick={[Function]}
    type="submit"
  >
     Submit 
  </button>
  <br />
</div>
`;
